js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js

```diff
@@ -625,8 +645,8 @@
 
             advancedTableContent.push(
               window.modalView.createTextEntry(
-                'new-min-replication-factor',
-                'Mininum replication factor',
+                'new-write-concern',
+                'Write concern',
```

Comment:
again, visible change?

So in this way our API names and the WebUI names differ.
I do not really like this...